app/app.vue

```diff
@@ -22,7 +22,18 @@
         An error occured, please look at Nuxt.js terminal.
       </h3>
     </div>
+    <div v-else-if="preventReload" class="reload_prevented">
+      <h3 class="hasErrors">Failed to show Nuxt.js app after {{ maxReloadCount }} reloads</h3>
+      <p>
+        Your Nuxt.js app could not be shown after {{ maxReloadCount }} reloads
```

Comment:
This is somehow repeated and not useful for user

Reply:
Have shortened the line in the paragraph, some repetition seems to be unavoidable? Or do you have a better suggestion?